crates/goose-mcp/src/developer/mod.rs

```diff
@@ -507,7 +507,8 @@ impl DeveloperRouter {
             .await
             .map_err(|e| ToolError::ExecutionError(e.to_string()))?;
 
-        let output_str = String::from_utf8_lossy(&output.stdout);
+        let stdout_str = String::from_utf8_lossy(&output.stdout);
+        let output_str = stdout_str;
```

Comment:
not sure what this change is doing?

Reply:
yeah, for some reason on windows it just had nothing in response and this works. Don't ask my why